### PR TITLE
onboarding description

### PR DESCRIPTION
**Pull request by sibirrer**
this is a draft for onboarding guidelines. I added a citation.rst file and made links to the contributing site.

### CODE REVIEW

File: CITATION.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Citation Guidelines
+===================
+
+We strive for open-source and reproducible science and all of our code is open-source available here.
+If you use part of this package for your research, please link to this repository.
+We plan to have several publications an revise these guidelines.
```
Comment (aprajita) [inline CITATION.rst:6]: and will revise these guidelines

File: ONBORDING.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+the two LSST Science collaborations:
+
+- `Strong Lensing Science Collaboration (SLSC) <https://sites.google.com/view/lsst-stronglensing?pli=1>`_
+- `Dark Energy Science Collaboration (DESC) <https://lsstdesc.org>`_
+
+
+link to pipeline overview (read only):
+https://docs.google.com/document/d/1dlwi7vP0jIL_miOF8VRiWd_0sALR4tklZlIIPuBJz9U/edit?usp=sharing
+
+link to template for sofware project
```
Comment (aprajita) [inline ONBORDING.rst]: move this up to line 11

File: ONBORDING.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+link to ICA agreement (upon request)
+
+
+How to contribute:
+
+You can also follow the designated Slack channel on lsstc.slack.com  #strong-lensing
```
Comment (aprajita) [inline ONBORDING.rst]: Add 
For pipeline specific discussion, please join #sl-pipeline and #lensed-sn-integration

File: ONBOARDING.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- `Strong Lensing Science Collaboration (SLSC) <https://sites.google.com/view/lsst-stronglensing?pli=1>`_
+- `Dark Energy Science Collaboration (DESC) <https://lsstdesc.org>`_
+
+You need to sign up for either one or both Science Collaboration to have access to LSST resources, communications and the Slack channel.
+We encourage you to sign up for both science collaborations.
+
+
+We have setup a document outlining how the LSST strong lensing simulation infrastructure could look like
+and what principles to follow to develop `here <https://docs.google.com/document/d/1dlwi7vP0jIL_miOF8VRiWd_0sALR4tklZlIIPuBJz9U/edit?usp=sharing>`_.
```
Comment (aprajita) [inline ONBOARDING.rst:16]: Can we add the generic template that i extracted from your sims doc so it'ss here for others to use as their templatess for elements of the pipeline? https://docs.google.com/document/d/1aCyo0D9XF48VNT7l8OvGBgxP3V4SLRlPxAJSrm2-ubg/edit

File: ONBOARDING.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+..
+    link to template for software project
+    link to ICA agreement (upon request)
+
+Mailing lists, Slack channels and telecon schedules are listed on the `confluence page <https://confluence.slac.stanford.edu/display/LSSTDESC/Topical+Team%3A+Strong+Lensing>`_ (need access).
+
+
+
+.. _Contributing: https://sim-pipeline.readthedocs.io/en/latest/contributing.html
```
Comment (aprajita) [inline ONBOARDING.rst:29]: should we add some coding guidleines?

**Review by aprajita — COMMENTED** [review]
hi @sibirrer the onboarding page looks good as do the contributor guidelines which are super helpful.  I made a few minor comments here. Not sure if I did it right.

**Comment by sibirrer** [comment]
> hi @sibirrer the onboarding page looks good as do the contributor guidelines which are super helpful. I made a few minor comments here. Not sure if I did it right.

Thank you very much @aprajita ! Yes, I can see your comments.
Instead of individual comments you could also have started a 'review' and then either stated 'request changes', 'approve' or 'comment'. Some comments are based on older code (or that I have not pushed before). I go through them and make the edits.

**Comment by sibirrer** [comment]
@aprajita I tried to make your requested changes. Can you approve the pull request (you should be able to do so). If you want, you can also merge it :) Thank you!

**Review by aprajita — APPROVED** [review]
Approved

File: AUTHORS.rst [hunk 6]
```diff
@@ -4,8 +4,8 @@ Credits
...
 This is a joint LSST-DESC and LSST-SLSC pipeline project.
 
-Contributors
-------------
+Contributors (alphabetical)
+---------------------------
 
 * Simon Birrer `sibirrer <https://github.com/sibirrer/>`_
-
+* Aprajita Verma `sibirrer <https://github.com/aprajita/>`_
```
Comment (aprajita) [inline AUTHORS.rst:11]: sorry - another one 1 - sibirrer after my name to aprajita

**Review by aprajita — CHANGES_REQUESTED** [review]

File: AUTHORS.rst (hunk 6 above)
Comment (sibirrer) [reply AUTHORS.rst]: uff, good catch! That's what reviews are for :) thank you very much, just updated it

**Pull request merged**